AirCasting/AuthViews/CreateAccountView.swift

```diff
@@ -26,7 +27,7 @@ struct CreateAccountView: View {
         GeometryReader { geometry in
             ScrollView {
                 VStack(spacing: 50) {
-                    if ((UserDefaults.standard.value(forKey: "onBoardingKey") != nil) == true) {
+                    if ((UserDefaults.standard.value(forKey: "hasEverLoggedIn") != nil) == true) {
```

Comment:
I know it's been here before, but I'd really like this to be handled by some protocolized object being injected into this view instead of using bare `UserDefaults`